build-system/npm-publish/write-package-files.js

```diff
@@ -164,6 +165,7 @@ async function writePackageJson() {
       preact: '^10.2.1',
       react: '^17.0.0',
     },
+    ...(useBentoCore ? {dependencies: {'@bentoproject/core': `tbd`}} : {}),
   };
 
```

Comment:
This seems unnecessarily complicated to read
```suggestion
  };
  if (useBentoCore) {
    json.dependencies = {'@bentoproject/core': `tbd`};
  }
```

Reply:
updated